src/ActionsInterface.php

```diff
@@ -33,4 +33,13 @@ public function getReviews(array $options = []);
      * @return mixed
      */
     public function findReview(string $reviewId);
+
+    /**
+     * Find reviews by $options.
+     *
+     * @param array $options
+     *
+     * @return mixed
+     */
+    public function findReviews(array $options);
```

Comment:
I would prefer to call this `getReviews`, but happy to discuss it if you prefer this